### DISCUSSION TITLE
Is reproject_match average resampling method similar to area weighted average?

### DISCUSSION THREAD

**Discussion by hamiddashti** [corteva/rioxarray#639] [Q&A] [answered]
So I have a  30 * 30 NDVI data (source image EPSG 102001) and I want to regrid it to 1 * 1 degree ( target image EPSG 4326). 

Is the `reproject_match` function with `average resampling` first reprojecting the source image to 4326, and then taking the area-weighted average, or is it first taking a simple average of all source pixels within a coarser pixel of the target image, then reprojecting it to EPSG 4326? My concern is whether the average resampling method used in reproject_match takes into account the source pixel's area when computing the average. 

Thanks

**Comment by snowman2** [comment, marked as answer]
That is a great question. Here is the documentation on the resampling methods: https://rasterio.readthedocs.io/en/latest/api/rasterio.enums.html#rasterio.enums.Resampling

It mentions using the weighted average, so I would assume that it does the area weighted average. It uses GDAL to perform the calculations, so you will likely get a better answer here: https://lists.osgeo.org/mailman/listinfo/gdal-dev

**Reply by hamiddashti** [reply]
Sounds it is takes area into account. Thanks!
